components/raftstore-v2/src/operation/command/admin/compact_log.rs

```diff
@@ -113,7 +113,7 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> PeerFsmDelegate<'a, EK, ER,
 
 impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     // Mirrors v1::on_raft_gc_log_tick.
-    fn maybe_propose_compact_log<T>(
+    fn maybe_propose_compact_log<T: Transport>(
```

Comment:
why change it?

Reply:
it calls on_admin_command, which needs to sends msgs now.